pkg/porter/credentials_test.go

```diff
@@ -315,7 +320,8 @@ func TestCredentialsEditEditorPathWithArgument(t *testing.T) {
 
 	p.Setenv("SHELL", "something")
 	p.Setenv("EDITOR", "C:\\Program Files\\Visual Studio Code\\code.exe --wait")
-	p.Setenv(test.ExpectedCommandEnv, "something -c C:\\Program Files\\Visual Studio Code\\code.exe --wait "+filepath.Join(os.TempDir(), "porter-kool-kreds.yaml"))
+	p.Setenv(test.ExpectedCommandEnv, "something -c C:\\Program Files\\Visual Studio Code\\code.exe --wait "+filepath.Join(os.TempDir(), "porter-kool-kreds.yaml")+
+		"\nsomething /C C:\\Program Files\\Visual Studio Code\\code.exe --wait "+filepath.Join(os.TempDir(), "porter-kool-kreds.yaml"))
```

Comment:
the formatting on these lines is hard to read consider using a string literal https://go.dev/ref/spec#String_literals and using `sprintf`